src/undici_version.h

```diff
@@ -2,5 +2,5 @@
 // Refer to tools/dep_updaters/update-undici.sh
 #ifndef SRC_UNDICI_VERSION_H_
 #define SRC_UNDICI_VERSION_H_
-#define UNDICI_VERSION "7.5.0"
+#define UNDICI_VERSION "6.21.2"
```

Comment:
That's not an update, more of a downgrade. @nodejs/undici should we close this?

Reply:
I guess this version was picked because https://github.com/nodejs/undici/releases/latest points to https://github.com/nodejs/undici/releases/tag/v6.21.2?

Reply:
This might be a small issue when releasing the last v6 version.

Suggest to drop the PR.

cc: @mcollina

Reply:
I've fixed it.